.github/workflows/create-release-on-tag.yml

```
@@ -22,13 +22,15 @@ jobs:
           tag_name: ${{ github.ref }}
           release_name: PXF Version ${{ github.ref }}
           body: |
-            ## 6.8.0 (09/25/2023)
+            ## 6.9.0 (12/04/2023)
 
             ### Enhancements:
 
-            - [#1013](https://github.com/greenplum-db/pxf/pull/1013) Bumped Azure Storage dependency to 5.5.0
-            - [#1018](https://github.com/greenplum-db/pxf/pull/1018) Add pxf.service.kerberos.ticket-renew-window option to pxf-site.xml
-            - [#1019](https://github.com/greenplum-db/pxf/pull/1019) Add pushdown of NUMERIC and handling of CHAR and VARCHAR predicates for JDBC profile
+            - [#1035](https://github.com/greenplum-db/pxf/pull/1035) Upgrade snappy-java to 1.1.10.4
+            - [#1040](https://github.com/greenplum-db/pxf/pull/1040) Bump golang.org/x/net from 0.7.0 to 0.17.0 in /cli
+            - [#1044](https://github.com/greenplum-db/pxf/pull/1044) Upgrade Go toolchain to 1.21.3
+            - [#1047](https://github.com/greenplum-db/pxf/pull/1047) Fix pxf register command
```

Should this be a bug fix?

---

As I understand, this fix was because we were failing silently before and are now erroring out appropriately. To me, this felt more like an improvement in user experience rather than a bug fix.